src/CardList/index.js

```diff
@@ -117,22 +132,60 @@ export default class ImageList extends Component {
   }
 
   render() {
-    let { cardLayout } = this.props
+    let { cardLayout, searchBar, items } = this.props
+    const { currentQuery } = this.state
     let wrap = [styles.wrap]
 
+    const newItems = items.filter((itm) => {
```

Comment:
Another idea that just occurred to me for code cleanliness - you could refactor this filtering into a function outside the render function. Another minor cleanliness point...